samples/Samples.h

```diff
@@ -80,7 +80,7 @@ typedef struct {
     BOOL enableFileLogging;
     UINT64 customData;
     PSampleStreamingSession sampleStreamingSessionList[DEFAULT_MAX_CONCURRENT_STREAMING_SESSION];
-    UINT32 streamingSessionCount;
+    volatile SIZE_T streamingSessionCount;
```

Comment:
See if you really need an atomic here or all of the operations requiring the streamingSessionCount are actually interlocked and you don't need an atomic.
In case you decide that you still need atomic, move the volatile SIZE_T to be the first first member of the stuct declaration to be machine word aligned even if it gets tight packed - we always declare the atomics first in order to assure they are machine word aligned, otherwise most hardware would generate bus faults